src/main/webapp/js/instructorFeedbackEdit/questionConstSum.es6

```diff
@@ -0,0 +1,71 @@
+/* global
+FEEDBACK_QUESTION_CONSTSUMPOINTS:false, FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHOPTION:false
+FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT:false, FEEDBACK_QUESTION_CONSTSUMOPTION:false
+FEEDBACK_QUESTION_NUMBEROFCHOICECREATED:false, FEEDBACK_QUESTION_CONSTSUMOPTIONTABLE:false
+*/
+
+function updateConstSumPointsValue(questionNum) {
+    if ($(`#${FEEDBACK_QUESTION_CONSTSUMPOINTS}-${questionNum}`).val() < 1) {
+        $(`#${FEEDBACK_QUESTION_CONSTSUMPOINTS}-${questionNum}`).val(1);
+    }
+    if ($(`#${FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHOPTION}-${questionNum}`).val() < 1) {
+        $(`#${FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHOPTION}-${questionNum}`).val(1);
+    }
+    if ($(`#${FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT}-${questionNum}`).val() < 1) {
+        $(`#${FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT}-${questionNum}`).val(1);
+    }
+}
+
+function addConstSumOption(questionNum) {
+    const questionId = `#form_editquestion-${questionNum}`;
+
+    const curNumberOfChoiceCreated = parseInt($(`#${FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNum}`).val(), 10);
+
+    $(`<div class="margin-bottom-7px" id="constSumOptionRow-${curNumberOfChoiceCreated}-${questionNum}">`
```

Comment:
Just to make sure, by template literals you mean:
```javascript
$(`
<div ...>
    bla bla bla... ${variable} ${variable}
</div>
`).
```

I guess I could make it in this PR as well.